dev/test-studio/sanity.config.ts

```diff
@@ -157,6 +157,13 @@ export default defineConfig([
     tasks: {
       enabled: true,
     },
+    features: {
+      beta: {
+        treeArrayEditing: {
+          enabled: true,
+        },
+      },
+    },
```

Comment:
I think it would be nice to move this to the `sharedSettings` plugin so that it is enabled for every config/workspace!

```ts
const sharedSettings = definePlugin({
  // ...

  features: {
    beta: {
      treeArrayEditing: {
        enabled: true,
      },
    },
  },
})
```

Reply:
Done :)